tests/unit/app/core/test_pipeline.py

```diff
@@ -12,10 +12,9 @@ def test_generate_pipeline_ingest_input(data_db: Session):
 
     documents = generate_pipeline_ingest_input(data_db)
     assert len(documents) == 2
-    assert documents[0].name == "Fam1"
-    assert documents[0].import_id == "CCLW.executive.2.2"
-    assert documents[1].name == "Fam1"
-    assert documents[1].import_id == "CCLW.executive.1.2"
+    for d in documents:
+        assert d.name == "Fam1"
+        assert d.import_id in ["CCLW.executive.2.2", "CCLW.executive.1.2"]
```

Comment:
The logic here is wrong - as it would pass if it returned one of these documents twice

Reply:
(I corrected the logic in the PR that just got merged)

Reply:
Great spot, thanks Peter!